info.json

```diff
@@ -20,10 +20,7 @@
             "minVersion": null
         }
     ],
-    "prerequisite": [
-        "Configure the Virustotal Connector",
-        "Configure the Splunk Connector"
-    ],
+    "prerequisite": "-Configure the Virustotal Connector\n- Configure the Splunk Connector",
     "publisher": "Fortinet",
     "category": [],
     "certified": "true",
```

Comment:
This is not a certified SP. Please mark it false

Reply:
Also, please work with Rohit/Sujit to see what all other changes are needed in info.json and data.json as per new code changes in platform.